### ISSUE TITLE
Kubeflow v0.7.0 kfctl delete does not clean up all resources

### ISSUE THREAD

**Issue by supertetelman** [kubeflow/kubeflow#4486]
Similar to https://github.com/kubeflow/kubeflow/issues/3767, in the newest release (v0.7.0) when I issue a delete command I am still seeing namespaces left behind for `cert-manager`, `istio-system`, `knative-serving`, and most of the time `kubeflow` is getting stuck in a terminating state. I also see some CRD left behind.

I am running this on a freshly installed on-prem K8S v1.15 installed via Kubespray with rook-ceph installed as the storage.

Following all the steps listed here for install and deletion: https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/

Namely:
```
export CONFIG_FILE=${KF_DIR}/kfctl_k8s_istio.0.7.0.yaml
kfctl delete -f ${CONFIG_FILE} --delete_storage
```

**Comment by sshakeri** [comment]
I had a issue. Here is a [workaround](https://github.com/kubeflow/kubeflow/issues/4496).

**Comment by lanya16** [comment]
![image](https://user-images.githubusercontent.com/22021811/68640649-698cbc00-0543-11ea-80ef-6b8c9a8910d5.png)

**Comment by srinjay-paul** [comment]
Issue still exists in kfctl v1.0-rc.1

**Comment by Kyrremann** [comment]
It also exists in kfctl v1.0-0-g94c35cf.

**Comment by supertetelman** [comment]
I'm still having  some issues with `error evaluating kustomization manifest for cert-manager`, but for the most part the new deletion logic is working well and this issue is resolved as of v1.1 rc-1.
